core/banking-protocol/src/main/resources/migration/master.xml

```diff
@@ -6,4 +6,5 @@
     <include relativeToChangelogFile="true" file="migrations/0000-init-schema.xml"/>
     <include relativeToChangelogFile="true" file="migrations/0001-create-table-banks.xml"/>
     <include relativeToChangelogFile="true" file="migrations/0002-mock-data.xml"/>
+    <include relativeToChangelogFile="true" file="migrations/0003-create-table-bank-profile.xml"/>
```

Comment:
I would suggest joining 
migrations/0003-create-table-bank-profile.xml
to 
migrations/0001-create-table-banks.xml
as versioning is not that important for now

Reply:
ok